src/viam/ur/module/ur_arm.cpp

```diff
@@ -290,16 +336,12 @@ void URArm::trajectory_done_cb(const control::TrajectoryResult state) {
     VIAM_SDK_LOG(info) << "\033[1;32mtrajectory report: " << report << "\033[0m";
 }
 
-void URArm::reconfigure(const Dependencies&, const ResourceConfig& cfg) {
-    // extract relevant attributes from config
-    current_state_->host = find_config_attribute<std::string>(cfg, "host");
-    current_state_->speed.store(find_config_attribute<double>(cfg, "speed_degs_per_sec") * (M_PI / 180.0));
-    current_state_->acceleration.store(find_config_attribute<double>(cfg, "acceleration_degs_per_sec2") * (M_PI / 180.0));
-    try {
-        const std::lock_guard<std::mutex> guard{current_state_->output_csv_dir_path_mu};
-        current_state_->output_csv_dir_path = find_config_attribute<std::string>(cfg, "csv_output_path");
-    } catch (...) {  // NOLINT: TODO: What should actually happen if the attribute is missing?
-    }
+void URArm::reconfigure(const Dependencies& deps, const ResourceConfig& cfg) {
+    VIAM_SDK_LOG(warn) << "Reconfigure called: shutting down current state";
+    shutdown_();
```

Comment:
[q] do we need to worry about the arm apis getting called while this/startup is in progress?

Reply:
We do, yes. However, I was hoping to stack that work above this, because I think a bigger overhaul of locking is required.